### ISSUE TITLE
Getting "upgrade to 2.8x required" when using May 2.80 versions

### ISSUE THREAD

**Issue by foolshope** [xavier150/Blender-For-UnrealEngine-Addons#13]
Appears in both my previous 2019-05-07 version and today's (2019-05-17) 2.8 beta. I see the addon's latest commits are in March, but I can't seem to find earlier versions of 2.8 to test.

Great work on your addon so far!

**Comment by xavier150** [comment]
I just tried with today's version (2019-05-17) but I don't have the "upgrade to 2.8x required"
Can you send me the console log ? I will update it if I find the problem

Thanks !

**Comment by foolshope** [comment]
Thanks for the fast response! The addon shows up normally in the Preferences after installing, and it can be enabled with the tick box without an error, but nothing shows up in the 'N' sidebar. It says the 'upgrade required' thing right next to the Addon name and there's a warning triangle next to the Community icon.

Just to be sure, I deleted the Blender appdata folder and tried it again in a fresh version. This is what the log printed:

> found bundled python: C:\Program Files\Blender Foundation\blender-2.80.0-git.2a9214eaaf04-windows64\2.80\python
> addon_utils.disable: blender-for-unrealengine not disabled
> Modules Installed (blender-for-unrealengine) from 'F:\\Stuff\\Blender\\_addons\\Blender-For-UnrealEngine-Addons-master.zip' into 'C:\\Users\\username\\AppData\\Roaming\\Blender Foundation\\Blender\\2.80\\scripts\\addons'

I hope this isn't some embarrassing user error. Let me know if I can give any more details.

**Comment by foolshope** [comment]
I'm so sorry for wasting your time, this was an user error after all!

I tried to figure this out and did some testing and reading and turns out I've been installing addons wrong for years. I've always just gone with the 'clone or download' instead of downloading from 'releases'. I did it the correct way and it works perfectly.

I'll close this issue and chuck a fiver in your Blender Market as an apology (and thanks for great addon) :)
